### PR TITLE
Empty state logic and template

### PR DESCRIPTION
**Pull request by phamann**
- Add logic inside `functions/create-email/index.js` for empty state type
- Add empty template
- Add tests for generate email and empty state

### CODE REVIEW

**Comment by phamann** [comment]
hmm some funky rebasing going on here...

File: functions/create-email/test/index.spec.js [hunk 1]
```diff
@@ -8,19 +8,22 @@ const invokeApex = require('../../../test/lib/invoke-apex');
...
 mockery.registerMock('./lib/group-content', groupedFixture);
+mockery.registerMock('./lib/email/generate-email', generateEmailFixture);
+mockery.registerMock('./lib-shared/metrics', metricsFixture);
 mockery.registerMock('./lib-shared/metrics', metricsFixture);
 
 const func = require('../');
 const eventFixture = { uuid: uuid(), type: 'daily' };
 
-describe('Create email', () => {
+describe.only('Create email', () => {
```
Comment (keirog) [inline functions/create-email/test/index.spec.js]: ❗

Comment (phamann) [inline functions/create-email/test/index.spec.js]: Nice catch 😎

**Comment by keirog** [comment]
👍
